app.go

```diff
@@ -957,6 +957,11 @@ func (app *App) startupMessage(addr string, tls bool, pids string) {
 		host = "0.0.0.0"
 	}
 
+	if app.config.Network == NetworkTCP6 {
```

Comment:
i think we should only set this if the host is not explicitly included in the address used for the listener

please move it in the `if host == "" {` body

Reply:
okay. will do this. Thanks!